examples/52_create_table_with_exclusion_constraint.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Is it possible to give an example migration that doesn't require the `btree_gist` extension?

All the example migrations so far run against a vanilla Postgres - this is the first one that requires an extension.